view/passenger_input_frame.py

```
@@ -3,6 +3,7 @@
 '''
 
 import ttkbootstrap as ttk
+from ttkbootstrap.dialogs.dialogs import Messagebox
```

Ruff

`ttkbootstrap.dialogs.dialogs.Messagebox` imported but unused

Remove unused import: `ttkbootstrap.dialogs.dialogs.Messagebox`

(F401)